packages/server/src/heartbeat.ts

```diff
@@ -1,17 +1,26 @@
-import { EventTarget } from '@medplum/core';
+import { TypedEventTarget } from '@medplum/core';
+import { MedplumServerConfig } from './config';
 
-export const heartbeat = new EventTarget();
+export type HeartbeatEventMap = {
+  heartbeat: { type: 'heartbeat' };
+};
+
+export const heartbeat = new TypedEventTarget<HeartbeatEventMap>();
```

Comment:
Separate conversation, out of scope for this PR, but I wonder if we could combine `TypedEventTarget` and `EventTarget` into a single class to avoid the wrapping.  Not a big deal, we can discuss offline.

Reply:
Probably could, shouldn't be an issue